python/graphstorm/eval/__init__.py

```diff
@@ -23,8 +23,7 @@
 from .eval_func import SUPPORTED_REGRESSION_METRICS
 from .eval_func import SUPPORTED_LINK_PREDICTION_METRICS
 
-from .evaluator import GSgnnInstanceEvaluator
 from .evaluator import GSgnnLPEvaluator
-from .evaluator import GSgnnMrrLPEvaluator, GSgnnPerEtypeMrrLPEvaluator
+from .evaluator import GSgnnPerEtypeMrrLPEvaluator
```

Comment:
We also need to update the implementation of GSgnnPerEtypeMrrLPEvaluator

Reply:
Yes, did.